galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/commons/ServletErrorMessage.java

```diff
@@ -12,6 +12,9 @@ public enum ServletErrorMessage {
     GAL5004_ERROR_RETRIEVING_PAGE                   (5004,"E: Error retrieving page. Report the problem to your Galasa Ecosystem owner."),
     GAL5005_INVALID_QUERY_PARAM_NOT_INTEGER         (5005,"E: Error parsing the query parameter ''{0}'' in the request URL. Invalid value ''{1}''. Expecting an integer."),
     GAL5006_INVALID_QUERY_PARAM_DUPLICATES          (5006,"E: Error parsing the query parameters. Duplicate instances of query parameter ''{0}'' found in the request URL. Expecting only one."),
+
+    GAL5010_FROM_DATE_IS_REQUIRED                   (5010,"E: Error parsing the query parameters. from time is a mandatory field if no runname is supplied."),    
```

Comment:
Quotes missing around the fields. 'from' and 'runname'.

Reply:
Its results in a 400 Bad Request HTTP status code